.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: test
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  linux:
+    name: linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

v2 uses a soon-to-be-deprecated version of nodejs: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/